src/vs/workbench/contrib/debug/browser/repl.ts

```diff
@@ -354,10 +359,13 @@ export class Repl extends Panel implements IPrivateReplService, IHistoryNavigati
 			}
 			const lastElementVisible = this.tree.scrollTop + this.tree.renderHeight >= this.tree.scrollHeight;
 			this.tree.updateChildren().then(() => {
-				if (lastElementVisible) {
-					// Only scroll if we were scrolled all the way down before tree refreshed #10486
-					revealLastElement(this.tree);
-				}
+				this.replDelayedRender = dom.runAtThisOrScheduleAtNextAnimationFrame(() => {
+					this.replDelayedRender = null;
```

Comment:
This is basically introducing a timeout, which we always try to avoid.
Can we try to fix this issue in a cleaner way?